src/OrchardCore.Modules/OrchardCore.AdminMenu/AdminNodes/LinkAdminNodeNavigationBuilder.cs

```diff
@@ -67,20 +90,5 @@ public Task BuildNavigationAsync(MenuItem menuItem, NavigationBuilder builder, I
                 }
             });
         }
-
-        // Add adminNode's IconClass property values to menuItem.Classes.
-        // Add them with a prefix so that later the shape template can extract them to use them on a <i> tag.
-        private void AddIconPickerClassToLink(string iconClass, NavigationItemBuilder itemBuilder)
```

Comment:
While this method is indeed not used, its referenced here too: https://github.com/OrchardCMS/OrchardCore/blob/8cc817c13d12ec39f4dbdec6f9c71195c0c511c1/src/docs/reference/modules/AdminMenu/README.md?plain=1#L142

Reply:
Yes the doc is obsolete, I assume that you meant to just update the doc?

Reply:
If the method doesn't make sense then yes.

Reply:
Okay done